packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -267,7 +267,7 @@ function generateReleaseNoteSlug(node) {
   const date = node.frontmatter.date ? node.frontmatter.date.split('T')[0] : '';
   const title = node.frontmatter.title ? node.frontmatter.title : '';
 
-  const slug = slugify(`${date} ${title}`, { lower: true });
+  const slug = slugify(`${date} ${title}`, { lower: true, strict: true });
```

Comment:
I did add "strict" to the slugify to make sure e.g. not parantheses and other characters are in the URL to have them consistent with all existing URLs.